src/orion/algo/space.py

```diff
@@ -312,44 +310,32 @@ def _is_numeric(item):
 
 
 class Real(Dimension):
-    """Subclass of `Dimension` for representing real parameters.
+    """Search space dimension that can take on any real value.
 
-    Attributes
+    Parameters
     ----------
     name : str
-    type : str
-    prior : `scipy.stats.distributions.rv_generic`
-    shape : tuple
-       See Attributes of `Dimension`.
+    prior : str
+       See Parameters of `Dimension.__init__`.
+    args : list
+    kwargs : dict
+       See Parameters of `Dimension.__init__` for general.
+
+    Real kwargs (extra)
+    -------------------
     low : float
-       Constrain with a lower bound (inclusive), default ``-numpy.inf``.
-    high : float
-       Constrain with an upper bound (exclusive), default ``numpy.inf``.
+       Lower bound (inclusive), optional; default ``-numpy.inf``.
+    high : float:
+       Upper bound (inclusive), optional; default ``numpy.inf``.
+    precision : int
+        Precision, optional; default ``4``.
+    shape : tuple
+       Defines how many dimensions are packed in this `Dimension`.
+       Describes the shape of the corresponding tensor.
 
```

Comment:
Since the expected behavior is that the higher bound is exclusive, maybe we could add a quick rationale for why it's not the case.

Reply:
Do you mean in the docstring for the parameter `high` or as a note?

Reply:
I was thinking to include it in the class's docstring.